### ISSUE TITLE
Apps created from pointing at a download install fail with ADT 17

### ISSUE THREAD

**Issue by GoogleCodeExporter** [erhanselman/mobile-web-development-with-phonegap#76]
```
Workaround:

After project creation:
Right click on project name
New -> Folder -> libs
Copy from Cordova install lib/android/example/libs/cordova-*.jar to the new 
libs folder
Right click on the new cordova*.jar and Select Add to Build Path
Right click on the old cordova*.jar and Select Remove from Build Path

More details at 
http://stackoverflow.com/questions/9820675/gson-noclassdeffounderror-after-adt-a
nd-sdk-tools-update-to-v17
```

Original issue reported on code.google.com by `paul.beu...@gmail.com` on 12 Apr 2012 at 12:29

**Comment by GoogleCodeExporter** [comment]
```
Updating Build Path is optional. The Android build tools will find the library 
in libs
```

Original comment by `paul.beu...@gmail.com` on 12 Apr 2012 at 12:54

**Comment by GoogleCodeExporter** [comment]
```
Addressed in 1.2.9
```

Original comment by `paul.beu...@gmail.com` on 19 Apr 2012 at 12:56
- Changed state: **Fixed**
